templates/documentation/reference/video-source-already-uploaded.md

```diff
@@ -25,6 +25,6 @@ You can cut down on mistakes by using one of our clients. We offer clients for o
 - [Go](../sdks/api-clients/apivideo-go-client.md)
 - [C#](../sdks/api-clients/apivideo-csharp-client.md)
 - [Java](../sdks/api-clients/apivideo-java-client.md)
-- [iOS](../sdks/api-clients/apivideo-swift5-client.md)
+- [iSwiftOS](../sdks/api-clients/apivideo-swift5-client.md)
```

Comment:
It seems to be copy/paste issue here

Reply:
Ah yes... the brand new platform from Apple, iSwiftOS! 😬 Let me fix it, along with the Swift icons. Great catch!